backend/apps/vacancies/tests/test_currency.py

```diff
@@ -0,0 +1,116 @@
+from apps.vacancies.models import Currency
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+UserModel = get_user_model()
+
+
+class CurrencyTestCase(TestCase):
+
+    def setUp(self):
+        """Starts before each test"""
+
+        Currency.objects.create(
```

Comment:
Try using factories, instead of hardcoding values, where possible. And why are you using pk here, as your db is empty and it will be the only currency anyway.